site/content/en/docs/tasks/integrate_a_custom_job.md

```diff
@@ -0,0 +1,47 @@
+---
+title: "Integrate a custom Job"
+date: 2022-07-25
+weight: 8
+description: >
+  Integrate a custom Job with Kueue.
+---
+# Integrate a custom Job with Kueue
+
+This page shows how to integrate a custom Job with Kueue. 
+
+## What we need to build
+
+1. Configration
+2. Controller
+3. Webhook
+
+## Configure your custom Job
+
+Add your framework name to `.integrations.frameworks` in `config/components/manager/controller_manager_config.yaml`
```

Comment:
I think it would be nice to provide direct github links when providing the path. If we provide the path it means we need to maintain to keep in sync with code anyway.